src/components/ClusterManager.js

```diff
@@ -419,7 +422,13 @@ export const DeleteClusterModal = ({ cluster: { googleProject, clusterName }, on
     onDismiss,
     okButton: deleteCluster
   }, [
-    'Deleting the notebook runtime will stop all running notebooks and associated costs. You can recreate it later, which will take several minutes.',
+    p(['Deleting the notebook runtime will stop all running notebooks and associated costs. You can recreate it later, which will take several minutes.']),
+    span({ style: { fontWeight: 'bold' } }, 'NOTE: '),
+    'Deleting your runtime will also delete any installed packaged and files on the associated hard disk (e.g. input data or analysis outputs). To permanently save these files, ',
+    h(Link, {
```

Comment:
This link is hard to read against the orange background. Try adding `variant: 'light'`

Reply:
This one has a white background, but I changed the one in the new cluster modal!